core/lib/config/src/configs/da_client/eigen.rs

```diff
@@ -1,10 +1,49 @@
+use std::str::FromStr;
+
 use serde::Deserialize;
-use zksync_basic_types::secrets::PrivateKey;
+use zksync_basic_types::{secrets::PrivateKey, Address, H160};
+
+/// Default address of the EigenDA service manager contract deployed on Holesky.
+const DEFAULT_EIGENDA_SVC_MANAGER_ADDRESS: &str = "0xD4A7E1Bd8015057293f0D0A557088c286942e84b";
 
-#[derive(Clone, Debug, Default, PartialEq, Deserialize)]
+/// Configuration for the EigenDA remote disperser client.
+#[derive(Clone, Debug, PartialEq, Deserialize)]
 pub struct EigenConfig {
-    pub rpc_node_url: String,
-    pub inclusion_polling_interval_ms: u64,
+    /// URL of the Disperser RPC server
+    pub disperser_rpc: String,
+    /// Block height needed to reach in order to consider the blob finalized
+    /// a value less or equal to 0 means that the disperser will not wait for finalization
+    pub settlement_layer_confirmation_depth: u32,
+    /// URL of the Ethereum RPC server
+    pub eigenda_eth_rpc: Option<String>,
+    /// Address of the service manager contract
+    pub eigenda_svc_manager_address: Address,
+    /// Wait for the blob to be finalized before returning the response
+    pub wait_for_finalization: bool,
+    /// Authenticated dispersal
+    pub authenticated: bool,
+    /// Url to the file containing the G1 point used for KZG
+    pub g1_url: String,
+    /// Url to the file containing the G2 point used for KZG
+    pub g2_url: String,
+    /// Chain ID of the Ethereum network
+    pub chain_id: u64,
```

Comment:
- If `chain_id` is removed from the verifier config (see the other comment), this looks unused.
- If this is somehow still required, a dumb question: can this chain ID differ from the Ethereum network chain ID in [the genesis config](https://github.com/matter-labs/zksync-era/blob/79f65780993d12b63fa2ed36a129c4f8f66624f9/core/lib/config/src/configs/genesis.rs#L21) (i.e., the L1 network for the validium)? Or [the settlement network ID](https://github.com/matter-labs/zksync-era/blob/79f65780993d12b63fa2ed36a129c4f8f66624f9/core/lib/config/src/configs/gateway.rs#L35) from `GatewayChainId`? If this config param *is* redundant, it shouldn't be copied; instead, the existing param should be reused.

Reply:
Fixed here https://github.com/lambdaclass/zksync-era/pull/386